packages/vat-data/src/types.d.ts

```diff
@@ -5,7 +5,6 @@ import type {
   WeakMapStore,
   WeakSetStore,
 } from '@agoric/store';
-import { Context } from 'vm';
```

Comment:
How had that snuck in?

Reply:
https://github.com/Agoric/agoric-sdk/pull/4962/files#diff-4d09d78ca33d43e2584cfbf5adc00ed1caac51f1edbc10bae3d3c14320e56d30R8